migration-engine/connectors/sql-migration-connector/src/sql_schema_differ/sql_schema_differ_flavour/postgres.rs

```diff
@@ -59,6 +59,26 @@ impl SqlSchemaDifferFlavour for PostgresFlavour {
             .collect()
     }
 
+    /// Return potential `CreateEnum` steps.
+    fn create_enums(&self, differ: &SqlSchemaDiffer<'_>) -> Vec<CreateEnum> {
```

Comment:
Sigh... would be great to return traits from trait methods....

Reply:
Long term I don't want to return anything from there, we just pass an `&mut something` argument and the flavour pushes there.